test/library/metrics/query.go

```diff
@@ -28,29 +30,31 @@ import (
 func NewPrometheusClient(ctx context.Context, kclient kubernetes.Interface, rc routeclient.Interface) (prometheusv1.API, error) {
 	_, err := kclient.CoreV1().Services("openshift-monitoring").Get(ctx, "prometheus-k8s", metav1.GetOptions{})
```

Comment:
should we create a const for namespace as well since used in manu places?

Reply:
I'd defer to a follow-up PR since it's not directly related to the bug.